Changelog.md

```diff
@@ -1,6 +1,42 @@
 ## Gazebo Msgs 11.x
 
-### Gazebo Msgs 11.x.x (202X-XX-XX)
+### Gazebo Msgs 11.0.0 (2024-08-26)
+
+Changes since 13.4.0:
+
+1. Improve migration note on SphericalCoordinates
+    * [Pull request #452](https://github.com/gazebosim/gz-msgs/pull/452)
+
+1. Deprecate LOCAL2 in SphericalCoordinates
+    * [Pull request #451](https://github.com/gazebosim/gz-msgs/pull/451)
```

Comment:
Consolidate

Reply:
Grouped in [5832a2d](https://github.com/gazebosim/gz-msgs/pull/455/commits/5832a2d4a599e8e4dcd87e4093350fb4e0866ee2)